cookbooks/fb_users/resources/default.rb

```diff
@@ -120,6 +122,15 @@
       password pass if pass
       action :create
     end
+
+    if manage_homedir
+      directory homedir do
+        owner mapinfo['uid']
+        group ::FB::Users::GID_MAP[pgroup]['gid'].to_i
```

Comment:
as mentioned before, this should probably be `info['homedir_group'] || ::FB::Users::GID_MAP[pgroup]['gid'].to_i`